### PR TITLE
SCALRCORE-16736: Create a simple CI pipeline for linting/unit tests

### PR DESCRIPTION
**Pull request by vmotso**
SCALRCORE-16736

### Changelog
* [ ] I have updated CHANGELOG.md

### Documentation
* [ ] I have updated resource/data source documentation in [website/docs](https://github.com/Scalr/terraform-provider-scalr/tree/develop/website/docs)

### State
* [ ] My changes affect the state
* [ ] I have included a state migration and a unit test for it

### CODE REVIEW

File: .github/workflows/default.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+name: default
```
Comment (vmotso) [inline .github/workflows/default.yml:1]: @lbn  I have doubts about the pipeline, jobs, steps naming. Maybe you can suggest something.

File: .github/workflows/default.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+on: push
+
+jobs:
+  lint:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up go 1.11.4
```
Comment (lbn) [inline .github/workflows/default.yml:11]: Readme says 1.13.

File: .github/workflows/default.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        uses: actions/setup-go@v2
+        with:
+          go-version: "1.11.4"
+      - name: fmt
+        run: test -z $(gofmt -l .)
```
Comment (lbn) [inline .github/workflows/default.yml:16]: golangci-lint can also run this checker. [Just need to enable it explicitly](https://golangci-lint.run/usage/linters#disabled-by-default-linters--e--enable)

**Review by lbn — CHANGES_REQUESTED** [review]

File: .golangci.yml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+run:
+  tests: false
+linters:
+  enable:
+    - gofmt
+  disable:
+    - errcheck
+issues:
+  exclude-rules:
```
Comment (vmotso) [inline .golangci.yml:9]: To ensure all checks just skips tests related files.

**Review by lbn — APPROVED** [review]

**Pull request merged**